test.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
```

Comment:
Ideally this should produce a clean output from [ShellCheck](https://www.shellcheck.net/) (you can also run it locally).